common/test/acceptance/pages/lms/learner_profile.py

```diff
@@ -165,3 +167,93 @@ def age_limit_message_present(self):
         """
         self.wait_for_ajax()
         return self.q(css='#u-field-message-account_privacy').visible
+
+    @property
+    def profile_has_default_image(self):
+        """
+        Return list of visible fields.
+        """
+        self.wait_for_field('image')
+        default_links = self.q(css='.image-frame').attrs('src')
+        return 'default-profile' in default_links[0] if default_links else False
+
+    def mouse_hover(self, element):
+        """
+        Mouse over on given element.
+        """
+        mouse_hover_action = ActionChains(self.browser).move_to_element(element)
+        mouse_hover_action.perform()
+
+    @property
+    def profile_has_image_with_public_access(self):
+        """
+        Check if image is present with remove/upload access.
+        """
+        self.wait_for_field('image')
+
+        self.mouse_hover(self.browser.find_element_by_css_selector('.image-wrapper'))
+        self.wait_for_element_visibility('.u-field-upload-button', "upload button is visible")
+        return self.q(css='.u-field-upload-button').visible
+
+    @property
+    def profile_has_image_with_private_access(self):
+        """
+        Check if image is present with remove/upload access.
+        """
+        self.wait_for_field('image')
+        return self.q(css='.u-field-upload-button').visible
+
+    def _upload_file(self, filename):
+        """
+        Helper method to upload a image file.
+        """
+        self.wait_for_element_visibility('.u-field-upload-button', "upload button is visible")
+        file_path = InstructorDashboardPage.get_asset_path(filename)
+
+        # make the elements visible.
+        self.browser.execute_script('$(".u-field-upload-button").css("opacity",1);')
+        self.browser.execute_script('$(".upload-button-input").css("opacity",1);')
+
+        self.wait_for_element_visibility('.upload-button-input', "upload button is visible")
+
+        self.browser.execute_script('$(".upload-submit").show();')
+        self.q(css='.upload-button-input').results[0].send_keys(file_path)
+
+        self.q(css='.upload-submit').first.click()
+
+        self.wait_for_ajax()
+
+    def upload_correct_image_file(self, filename):
```

Comment:
Nit: this function doesn't seem necessary. I would probably just make ``_upload_file`` public.